pkg/cmd/cli/backuplocation/create.go

```diff
@@ -53,17 +54,23 @@ func NewCreateCommand(f client.Factory, use string) *cobra.Command {
 }
 
 type CreateOptions struct {
-	Name     string
-	Provider string
-	Bucket   string
-	Prefix   string
-	Config   flag.Map
-	Labels   flag.Map
+	Name       string
+	Provider   string
+	Bucket     string
+	Prefix     string
+	Config     flag.Map
+	Labels     flag.Map
+	AccessMode *flag.Enum
 }
 
 func NewCreateOptions() *CreateOptions {
 	return &CreateOptions{
 		Config: flag.NewMap(),
+		AccessMode: flag.NewEnum(
+			string(velerov1api.BackupStorageLocationAccessModeReadWrite),
+			string(velerov1api.BackupStorageLocationAccessModeReadWrite),
```

Comment:
This is a dup of the prior one.

Reply:
the first arg is the default value, then you provide a list of all the possible options, so it needs to be specified twice.